client/src/App.js

```diff
@@ -170,6 +179,7 @@ class App extends Component {
         this.setState({
           role: role,
           userDetails: profile,
+          verified: profile.verified,
```

Comment:
So apologies for being unclear about this.
`verified` would mean email verified. Once user's sign up they would get an email with a verification link they would have to click (to be implemented) to confirm their emails.
`approved` is the field we should be using!